AutoImplement.nuspec

```diff
@@ -22,7 +22,7 @@
     <copyright>Copyright 2018</copyright>
   </metadata>
   <files>
-    <file src="artifacts\AutoImplement\bin\Release\AutoImplement.exe" target="" />
+    <file src="artifacts\AutoImplement\bin\Release\AutoImplement.exe" target="content\libs" />
```

Comment:
is `content\libs` the best possible place for this? I would think that `content\tools` might be more appropriate. Libs/lib is usually reserved for libraries that you want to reference (such as the nuget target="lib" used for System.Delegation). "Tools" is probably a better name for something called as a pre-build step that isn't actually referenced by the project.

Reply:
I only selected libs because that is the path that is currently suggested in the wiki. I agree that tools makes more sense.

Reply:
Good point. While updating the wiki, I went ahead and updated that path based on the current expected install scenario. I can update it again once this gets merged in.